proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,571 @@
+   Mitglied mit schriftlicher Vollmacht vertreten lassen.
+7. Die Teilnahme an der Mitgliederversammlung und die Stimmabgabe müssen auch
+   auf elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann
+   eine Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.
+
+## 7. Aufgaben der Mitgliederversammlung
+
+1. Die Mitgliederversammlung ist das höchste beschlussfassende Organ und ist
+   für alle Aufgaben zuständig, sofern diese nicht anderen zugewiesen
+   wurden.
+2. Sie wählt den Vorstand aus dem Kreis der Kernmitgliedern und kann
+   Vorstandsmitglieder abwählen.
+3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
+4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
+   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
+5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
+6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
+   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
+   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
+   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
+   beim Verein angestellt ist, um die Buchführung einschließlich
+   Jahresabschluss zu prüfen und über das Ergebnis vor der
+   Mitgliederversammlung zu berichten. Der Rechnungsprüfer hat Zugang zu allen
+   Buchungs- und Rechnungsunterlagen des Vereins.
+7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
+   Krediten.
+
+## 8. Der Vorstand
+
+1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
+   Schatzmeister, die für eine Amtszeit von 3 Jahren gewählt werden.
+2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen wird durch eine
+   Geschäftsordnung des Vorstands geregelt. Der Vorsitzende ist dabei
+   für die Verfolgung des Vereinszwecks, die Einhaltung der Satzung und die
+   Delegation von Aufgaben verantwortlich. Der zweite Vorsitzende vertritt den
+   Vorsitzenden bei dessen Abwesenheit und überprüft die Beschlüsse des
+   Vorstands und der Mitgliederversammlungen. Der Schatzmeister verwaltet die
+   Finanzen des Vereins, führt die Kassenbücher und die Mitgliederliste.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die
+   Beschlüsse der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand ein kommissarisches Vorstandsmitglied. Das
+   kommissarische Vorstandsmitglied bleibt bis zur nächsten
+   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
```

Comment:
Ich glaube nicht, dass es rechtlich möglich ist, ein amtierendes Vorstandsmitglied aus dem Verein auszuschließen. Dies kann eigentlich nur die Mitgliederversammlung. Die kann man natürlich außerordentlich einberufen.

https://www.experto.de/praxistipps/vereinsausschluss-eines-vorstandsmitglieds-so-gehen-sie-vor.html

Reply:
Ich gehe davon aus das alles in Ordnung ist, da der Ausschluss nur durch die Kernmitglieder bestimmt werden kann, die auch für die Wahl des Vorstands zuständig sind. Oder hast Du einen Änderungsvorschlag?

Reply:
Die im verlinken Artikel erwähnte BGH-Rechstsprechung besagt, dass nur das gleiche Gremium, dass ein Vorstandsmitglied ernennt, es auch wieder abberufen kann. Also in unserem Fall nur die Mitgliederversammlung.

Der Fall kann also von Gesetzes wegen nicht vorkommen. Daher bitte 6. komplett streichen.

Reply:
I needed to translate this in several translation apps to get the correct meaning, but I suppose it's correct that the organ that gives a mandate can take the mandate back, and only that organ.

Reply:
sorry for this conversation only being in german, but its about german legal text so unless you can read the legal text in question precisely, it doesn't make much sense to participate in the conversation anyways.

Reply:
kein problem, mein Deutsch ist mehr als gut genug, aber bestimmte begriffe sind etwas undeutlich. Ich habe keinen deutschen juridischen hintergrund. Aber viele sachen sind das selbsten in Deutschland wie in Belgien oder Frankreich. So ich verstehe die kerngedachte sehr gut. :-)

Reply:
OK my interpretation was that the same Group of people have decide in our case they can decide in an internet vote or the general assembly. I am hesitating to delete that paragraph without replacement, because it is also written in the linked article that the Satzung applies and the court jumps in in unclear cases. 

Let's assume there is a twist with a board member that is doing harm to the association and quick solution is required to stop that. 

According to the current Satzung we can withdraw them form the association with §4 (5) by an internet vote which takes two weeks and after the person had the chance to rectify their behaviour.

The alternative would be a extraordinary General Assembly this requires an internet vote to decide for it (2 weeks) and a 6 weeks invitation period. Which is a long time for doing more harm.  
 
What do you think? What suits to us?

Reply:
You can not withdraw a board member from an e.V. without a general assembly. This is not allowed by law and can not be overruled by the Satzung.

Reply:
We can add an article about the board itself, they're unpair (3) , if 2 of them decide that the third of them is harming the roganization they can take over his responsabilities right away. The board contains the elected representatives from the members, so they can act fast, but this needs to be confirmed with a general assembly's decision in 2 weeks.

Reply:
it's not to withdraw a member of the board it's to block possible harmful begavious/decisions

Reply:
According to the law:
1. You are board member until the court changes the register - and they do this only based on the protocol of a general assembly
2. A majority vote of the board is needed for decisions of the board
3. Each board member is allowed to speak in the name of the e.V. in public - you can not restrict this

Reply:
'Each board member is allowed to speak in the name of the e.V. in public ' is not the same as 'Each board member can make commitments or engagements in the name of the e.V'
In the bylaw document can be stipulated that an oral commitment of one of the board members has only value when it's confirmed by a written declaration (digitally) signed by all board members.

Reply:
To continue on Daniels use case: one board member having strange thoughts can't harm the organization that much.
2 or 3 stepping of the right is a complete different dangerous situation.

Reply:
Ok, this also confirms: 
https://www.iww.de/vb/archiv/vereinsrecht-kein-ausschluss-von-vorstandsmitgliedern-durch-vorstand-f17976#:~:text=093187)-,Vorstandsmitglieder%20k%C3%B6nnen%20nicht%20durch%20den%20Vorstand%20vom%20Verein%20ausgeschlossen%20werden,f%C3%BCr%20den%20Vereinsausschluss%20zust%C3%A4ndig%20ist.

I will delete the 6. 

I will also refine the right to speak up. It shall be written and by a deadline of 2 weeks. OK?